### ISSUE TITLE
BEiT3 Warnings when fine-tuning on the VQAv2 dataset

### ISSUE THREAD

**Issue by lrdhaha** [microsoft/unilm#1048]
Before describing the problem I'll start with some background:
I used the official setup guide, which is to install dependent packages on the corresponding docker environment
The fine-tuning training command is consistent with the official one, see the figure below, the only difference is that the batchsize is not 16 but 4.
python -m torch.distributed.launch --nproc_per_node=8 run_beit3_finetuning.py --model beit3_base_patch16_480 --input_size 480 --task vqav2 --batch_size 4 --layer_decay 1.0  --lr 3e-5  --update_freq 1   --epochs 10  --warmup_epochs 1  --drop_path 0.1  --sentencepiece_model /home/data/unilm/beit3/pretrain_model/beit3.spm --finetune /home/data/unilm/beit3/pretrain_model/beit3_base_patch16_480_vqa.pth --data_path /home/data/data/coco_vqav2 --output_dir /home/data/unilm/beit3/workdirs/model --log_dir /home/data/unilm/beit3/workdirs/model/log --weight_decay 0.01  --seed 42  --save_ckpt_freq 5  --task_head_lr_weight 20  --opt_betas 0.9 0.98 --randaug 

The following warnings and problems occurred when I finetune on the VQAv2 dataset according to the official guide:
1. The value of grad norm is nan, is this normal?
![1680080158011](https://user-images.githubusercontent.com/99938170/228481715-50203591-cb5d-4f30-8ae0-2b907e4a06bf.jpg)

2. Warning: NaN or Inf found in input tensor. What is the reason for this warning?
![1680080261874](https://user-images.githubusercontent.com/99938170/228482121-952321e4-1bb1-48a8-80dc-aed047d82303.jpg)

3. The loss is very unstable during fine-tuning. I think this should be normal, but I am not sure
![1680080329728](https://user-images.githubusercontent.com/99938170/228482447-0fc94ce1-44b1-49a7-ae68-b95e4a1d396c.jpg)
4. By the way, I would like to ask what is the meaning of the brackets in loss and gradnorm?
![1680080514485](https://user-images.githubusercontent.com/99938170/228483317-c6c07d73-1327-43fb-ba6b-881e8e5be5ac.jpg)

I just came into contact with this task recently, thank you for your patient guidance.

**Comment by addf400** [comment]
Q1 & Q2 & Q3 are normal. BEiT-3 code use [Automatic Mixed Precision package](https://pytorch.org/docs/stable/amp.html) for finetuning. 

“gradient scaling” multiplies the network’s loss(es) by a scale factor and invokes a backward pass on the scaled loss(es). If gradients are checked for infs/NaNs, the gradient scaler will reduce the scale factor. More details can be found at [this url](https://pytorch.org/docs/stable/amp.html#gradient-scaling).

Q4: the brackets in loss and gradnorm is the [smooth value](https://github.com/microsoft/unilm/blob/master/beit3/utils.py#L43) for this epoch.

**Comment by lc222** [comment]
why i can't find beit3_base_patch16_480 in timm?
timm                     0.4.12
torchscale               0.2.0

**Comment by 464hee** [comment]
> Q1 & Q2 & Q3 are normal. BEiT-3 code use [Automatic Mixed Precision package](https://pytorch.org/docs/stable/amp.html) for finetuning.
> 
> “gradient scaling” multiplies the network’s loss(es) by a scale factor and invokes a backward pass on the scaled loss(es). If gradients are checked for infs/NaNs, the gradient scaler will reduce the scale factor. More details can be found at [this url](https://pytorch.org/docs/stable/amp.html#gradient-scaling).
> 
> Q4: the brackets in loss and gradnorm is the [smooth value](https://github.com/microsoft/unilm/blob/master/beit3/utils.py#L43) for this epoch.

Looking forward to your reply, can't find beit-3 model in timm
